GridMvc.Site/Models/Grids/OrdersAjaxPagingGrid.cs

```diff
@@ -27,6 +27,7 @@ public AjaxGridPager(IGrid grid)
         public int CurrentPage { get; set; }
 
         public int? ItemsCountOverwrite { get; set; }
+        public bool? CustomPaging { get; set; }
```

Comment:
What is the difference in behavior of `false` and `null`? Could this be non-nullable?

Reply:
Is now a non-nullable